inst/rmarkdown/templates/measles_outbreak/skeleton/skeleton.Rmd

```diff
@@ -620,20 +620,20 @@ DATEVARS <- DATEVARS[-EMPTY_DATEVARS]
 linelist_cleaned <- linelist_cleaned %>%
   mutate(
     across(.cols = all_of(DATEVARS),
-           .fns = ~parsedate::parse_date(.x)))
+           .fns = ~ymd(parsedate::parse_date(.x))))
```

Comment:
parse_date() returns class POSIXct, not Date. Do I wrap in ymd() to make the subsequent code work more smoothly